CMakeLists.txt

```diff
@@ -101,7 +88,7 @@ option(WITH_OPTIM "Build with optimisation" ON)
 option(WITH_NEW_STRATEGIES "Use new strategies" ON)
 option(WITH_NATIVE_INSTRUCTIONS
     "Instruct the compiler to use the full instruction set on this host (gcc/clang -march=native)" OFF)
-if("${ARCH}" MATCHES "arm" OR "${ARCH}" MATCHES "aarch64")
+if(BASEARCH_ARM_FOUND)
     option(WITH_ACLE "Build with ACLE CRC" ON)
     option(WITH_NEON "Build with NEON intrinsics" ON)
 elseif("${ARCH}" MATCHES "s390x")
```

Comment:
Consider `BASEARCH_S390_FOUND AND "${ARCH}" MATCHES "s390x"`.. for clarity and a bit of speed.

Reply:
S360, but yeah, probably a good idea for clarity